besspin/cwesEvaluation/scoreTests.py

```diff
@@ -276,7 +302,7 @@ def tabulate(elements, vulClass, title, hasMultitaskScores):
     """
     table = []
 
-    headers = ["TEST","Score","Notes"]
+    headers = ["CWE","Score","Score","Notes"]
```

Comment:
Can you add a comment here explaining that one of the "Score" elements is removed later?  It confused me until I saw the code later down that removed it.

Reply:
Thanks. Added a comment.